merlin/systems/dag/ops/pytorch.py

```diff
@@ -93,6 +96,17 @@ def compute_output_schema(
         """
         return self.output_schema
 
+    def transform(self, col_selector: ColumnSelector, transformable: Transformable):
+        tensor_dict = {}
+        for column in transformable.columns:
+            tensor_dict[column] = torch.from_numpy(np.squeeze(transformable[column].values))
```

Comment:
Should we be doing this in another place? Should dict array have APIs for transforming to tensors of specific frameworks?

Reply:
I think it should eventually (soon, even) but since we don't have those yet, this seems reasonable for now